main/export-list.js

```diff
@@ -120,6 +121,23 @@ class ExportList {
       }
     }
 
+    if (!newExport.plugin.isConfigValid()) {
+      const result = dialog.showMessageBox({
+        type: 'error',
+        buttons: ['Configure', 'Cancel'],
+        defaultId: 0,
+        message: 'Error in plugin configuration',
+        detail: 'Review your plugin configuration for "now" to continue exporting',
```

Comment:
Slight tweaked. Better?

> Review the config for the `now` plugin to continue exporting

Reply:
Wow `now` was also hardcoded instead of using the pluginName 😆